FrontEnd/src/global.css

```diff
@@ -0,0 +1,44 @@
+ @import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```

Comment:
Do we need to import the `global.css` file to the root of our app?

Reply:
Yes, at least that's how I understood the task.

Reply:
I ask it since I do not see import `global.css` in the PR.

Reply:
I ask it since I do not see import `global.css` in the PR. Is the PR ready for review?

Reply:
Previously, this file was not used in our project and each of the developers wrote styles in their own way. After we add this file, we can use it in work. We can import it into index.css or better into index.js.

Reply:
Yes, that is the main point of having `global.css`. I still do not see the import of it in `App.js`

Reply:
Done.